template/iconComponent.ts

```diff
@@ -0,0 +1,25 @@
+import { TemplateProps } from '../interface/index';
+
+function componentTemplate ({fileName,data}:TemplateProps) {
+    const render = `
+// GENERATED BY ./scripts/iconScript.ts
+
+import * as React from 'react';
+import { IconProps, valuesMap } from '../helper';
+
+function Eos${fileName}({size = "m", color = "black", rotate = 0}: IconProps) {
+    const sizeString: string = size.toString()
```

Comment:
this is not needed, if it is a number, `.includes` will return false

Reply:
Everything else lgtm now

Reply:
it is throwing a type error because we are accepting both number and string as prop input but this expression is expecting a string
`Argument of type 'string | number' is not assignable to parameter of type 'string'.
  Type 'number' is not assignable to type 'string'.ts(2345)`